tests/utils/inputparams/testparams.go

```diff
@@ -53,6 +54,15 @@ func GetContainerNameWithTimeStamp(containerName string) string {
 	return containerName + "_container_" + strconv.FormatInt(time.Now().Unix(), 10)
 }
 
+func GetVolumeNameOfSize(size int) string {
+	const chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
I think we can go ahead with prefixes, but some test cases where we require exact sized volume names, we can make an exception to keep the util simple.